maintainers/maintainer-list.nix

```
@@ -6703,6 +6703,12 @@
     name = "Sebastian Neubauer";
     keys = [ { fingerprint = "2F93 661D AC17 EA98 A104  F780 ECC7 55EE 583C 1672"; } ];
   };
+  Flameopathic = {
+    email = "flameopathic@gmail.com";
+    github = "Flameopathic";
+    githubID = 64027365;
```

```suggestion
    githubId = 64027365;
```

Seems to be a typo, this is what is causing the CI failures.